drivers/virtio/virtio-serial.c

```diff
@@ -558,6 +562,15 @@ static int virtio_serial_probe(FAR struct virtio_device *vdev)
       goto err_with_init;
     }
 
+#ifdef CONFIG_VIRTIO_SERIAL_CONSOLE
+  if (!g_virtio_console)
+    {
+      DEBUGVERIFY(uart_register("/dev/console", &priv->udev));
```

Comment:
Should the `isconsole` flag be updated in uart driver?
https://github.com/apache/nuttx/blob/master/drivers/serial/uart_16550.c#L1729